src/components/Modals/Regions/PurchaseHistory/index.tsx

```diff
@@ -39,7 +39,7 @@ export const PurchaseHistoryModal = ({
     network,
     regionBegin,
     0,
-    20
+    1000
```

Comment:
Subscan supports pagination, we should utilize it.

Reply:
Yeah, that was the first approach I tried.
As the number of coretime purchases does not exceed 1000, I think it makes sense.

Reply:
Ok. If we have performance issues we can update it later then.